src/Octoshift/AzureApi.cs

```diff
@@ -49,14 +49,14 @@ private Uri GetServiceSasUriForBlob(BlobClient blobClient)
                 throw new InvalidOperationException("BlobClient object has not been authorized to generate shared key credentials. Verify --azure-storage-connection-key is valid and has proper permissions.");
             }
 
-            var sasBuilder = new BlobSasBuilder()
+            var sasBuilder = new BlobSasBuilder
             {
                 BlobContainerName = blobClient.GetParentBlobContainerClient().Name,
                 BlobName = blobClient.Name,
-                Resource = "b"  // Resource = "b" for blobs, "c" for containers
-            };
+                Resource = "b",  // Resource = "b" for blobs, "c" for containers
 
-            sasBuilder.ExpiresOn = DateTimeOffset.UtcNow.AddHours(AUTHORIZATION_TIMEOUT_IN_HOURS);
+                ExpiresOn = DateTimeOffset.UtcNow.AddHours(AUTHORIZATION_TIMEOUT_IN_HOURS)
+            };
```

Comment:
I wouldn't do this refactor as part of this PR.

Reply:
I wouldn't either. This was probably some formatting that happened automatically and I didn't notice it.

Good catch

Reply:
I wouldn't either. This was probably some formatting that happened automatically and I didn't notice it.